docs/references/configuration.rst

```diff
@@ -708,6 +708,22 @@ log-level
 
   Because currently there's no buffering for logging, the levels with minimal logging(``crit/error``) will increase throughput.
 
+.. _log-query:
+
+log-query
+---------
+
+  =============== =================================
+  **Type**        Boolean
+  **Default**     False
+  **Reloadable**  Y
+  **Environment** PGRST_LOG_QUERY
+  **In-Database** `pgrst.log_query`
```

Comment:
Hm, should this work in-db? The `log-level` currently doesn't work..

My initial reasoning for omitting `log-level` was to not make it easy to flood logs and cause malfunction on the PostgREST service. I believe these types of settings need to be really "visible" for the operator at the service level (needs to show in its systemd service or config file).